keyboards/choco60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2019 Naoto Takai
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define _BASE 0
```

Comment:
As I said on the Cocoa40 pull request, you could use an enum here in place of Lines 19-20:

```c
enum layer_names {
    _BASE,
    _FN,
};
```

I think it's less important to do when there are only two layers; I usually only insist on it when there are four layers or more.